sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -6,291 +6,129 @@
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Positive_Software_limit</EFDB_Name>
-          <Description>Azimuth_Positive_Software_limit</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Negative_Software_limit</EFDB_Name>
-          <Description>Azimuth_Negative_Software_limit</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Positive_Power_Off_switch</EFDB_Name>
-          <Description>Azimuth_Positive_Power_Off_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
-          <Count>1</Count>
-      </item>
-     <item>
-          <EFDB_Name>Azimuth_Negative_Power_Off_switch</EFDB_Name>
-          <Description>Azimuth_Negative_Power_Off_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>amp</Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>Azimuth_Positive_Directional_limit_switch</EFDB_Name>
-          <Description>Azimuth_Positive_Directional_limit_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <EFDB_Name>CCW_Current_2</EFDB_Name>
+          <Description>CCW_Current_2</Description>
+          <IDL_Type>double</IDL_Type>
+          <Units>amp</Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>Azimuth_Negative_Directional_limit_switch</EFDB_Name>
-          <Description>Azimuth_Negative_Directional_limit_switch</Description>
-          <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <EFDB_Name>sub_Rotator_Position</EFDB_Name>
+          <Description>sub_Rotator_Position</Description>
+          <IDL_Type>double</IDL_Type>
+          <Units>deg</Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>Azimuth_Positive_Adjustable_Software_limit</EFDB_Name>
-          <Description>Azimuth_Positive_Adjustable_Software_limit</Description>
+          <EFDB_Name>CCW_Positive_Directional_limit_switch</EFDB_Name>
+          <Description>CCW_Positive_Directional_limit_switch</Description>
           <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
           <Count>1</Count>
       </item>
      <item>
-          <EFDB_Name>Azimuth_Negative_Adjustable_Software_limit</EFDB_Name>
-          <Description>Azimuth_Negative_Adjustable_Software_limit</Description>
+          <EFDB_Name>CCW_Negative_Directional_limit_switch</EFDB_Name>
+          <Description>CCW_Negative_Directional_limit_switch</Description>
           <IDL_Type>boolean</IDL_Type>
-          <Units>unitless</Units>
+          <Units>   </Units>
```

Comment:
Units cannot be blank or just whitespace.  Please use `unitless` or `dimensionless`.